src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 30;
+  const SHORT_TERM_DISCOUNT = 20;
+  const ONE_DAY_PRICE = 40;
+
+  let DAY_PRICE = ONE_DAY_PRICE * days;
+
+  if (days >= LONG_TERM) {
+    DAY_PRICE -= LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    DAY_PRICE -= SHORT_TERM_DISCOUNT;
+  }
```

Comment:
The current logic will apply both discounts if the number of days is 7 or more, which is incorrect. The logic should be exclusive; if the number of days is 7 or more, only the LONG_TERM_DISCOUNT should be applied. If it is 3 or more but less than 7, only the SHORT_TERM_DISCOUNT should be applied.